dataplane/linux/vxlan_mgr.go

```diff
@@ -105,6 +105,9 @@ func newVXLANManager(
 		blackHoleProto = dpConfig.DeviceRouteProtocol
 	}
 
+	noencTarget := routetable.Target{Type: routetable.TargetTypeNoEncap}
+	bhTarget := routetable.Target{Type: routetable.TargetTypeBlackhole}
```

Comment:
I don't think we need these.  Simpler just to put `routetable.TargetType...` inline below.